src/oss/confluent.sh

```diff
@@ -1053,6 +1076,11 @@ connect_restart_command() {
     echo "Not implemented yet!"
 }
 
+dynamic_cli_command() {
+    local command ="${1}"
+    exec ./${command}-cli "$@"
```

Comment:
Relative path won't work in general. We can't assume anything about where the CLI is executed from. 

Useful directories for that are: `$confluent_bin` `$confluent_conf` and `$confluent_home`
`$confluent_current_dir` is where CLI stores its data.